versionInfo.gradle

```diff
@@ -1,5 +1,20 @@
 group = 'com.workday.warp'
 
+
+println project.findProperty("scalaVersion")
+
+
+
```

Comment:
Do we want documentation for gradle functions?

Reply:
yep ill add that